frontend/sentiment-dashboard/src/app/login/login.component.ts

```diff
@@ -41,13 +42,14 @@ export class LoginComponent implements OnInit {
 
   onSubmitLogin(value: any) {
     const username = value.username, password = value.password;
+    this.error = '';
     this.auth.signIn(username, password).subscribe(
       result => {
         this.auth.isAdmin().subscribe(val => this.auth.setAdmin(val));
         this.router.navigate(['/']);
       },
-      error => {
-        console.log(error);
+      err => {
+        this.error = 'Incorrect username or password.';
```

Comment:
Can we be sure that this error will always be due to an incorrect username or password? Seems like this might be a more general error values from the `auth.signIn` call...

Reply:
I changed it to use the error messages cognito provides so they're more clear

Reply:
Looks good - thanks!